docs/USAGE.md

```diff
@@ -27,50 +27,50 @@
 
 JSON-Schema-faker (or `jsf` for short) combines two things:
 
- * The [JSON-schema specification](http://json-schema.org/draft-04/json-schema-core.html), that defines what is the allowed content of a JSON document
- * Fake data generators, that are used to generate basic or complex data, conforming to the schema.
+- The [JSON-schema specification](http://json-schema.org/draft-04/json-schema-core.html), that defines what is the allowed content of a JSON document
+- Fake data generators, that are used to generate basic or complex data, conforming to the schema.
 
-> Since `v0.5.x` external generators are not longer bundled with `jsf`, however built-in defaults are shipped for all basic types and formats.
+> Since `v0.5.x` external generators are not longer bundled with `JSONSchemaFaker`, however built-in defaults are shipped for all basic types and formats.
 
 ## Example usage
 
 ```javascript
-import jsf from 'json-schema-faker';
+import { JSONSchemaFaker } from "json-schema-faker";
```

Comment:
This is the reason I updated this file. Using the named export instead of the default. 

I also changed `jsf` to `JSONSchemaFaker` to match the above code where necessary.

Reply:
Yeah, I think this is the best outcome, thank you!